README.md

```diff
@@ -3,20 +3,16 @@
 Coinbase Wallet SDK (formerly WalletLink) lets developers connect their dapps to Coinbase Wallet
 on both mobile web (for iOS and Android) and desktop:
 
-- **Mobile**: Users can connect to your mobile web dapp through a deeplink to the
-  dapp browser in Coinbase Wallet Mobile App.
+- **Mobile**: Users can connect to your mobile web dapp through a deeplink to the dapp browser in Coinbase Wallet Mobile App.
 
-- **Desktop**: Users can connect to your desktop app with a QR code in the
-  Coinbase Wallet Mobile App or with the Coinbase Wallet Chrome Extension.
+- **Desktop**: Users can connect to your desktop app with a QR code in the Coinbase Wallet Mobile App or with the Coinbase Wallet Chrome Extension.
```

Comment:
Should we also link out to the chrome extension store [page or mobile app](https://coinbase-wallet.onelink.me/q5Sx/fdb9b250)?

Reply:
Done!